src/Microsoft.ML.Transforms/PermutationFeatureImportanceExtensions.cs

```diff
@@ -159,4 +166,179 @@ public static class PermutationFeatureImportanceExtensions
                 auprc: a.Auprc - b.Auprc);
         }
     }
+
+    public sealed class MetricStatistics
+    {
+        /// <summary>
+        /// Get the mean value for the metric
+        /// </summary>
+        public double Mean => _statistic.Mean;
+
+        /// <summary>
+        /// Get the standard deviation for the metric
+        /// </summary>
+        public double StandardDeviation => ComputeStandardDeviation();
+
+        /// <summary>
+        /// Get the standard error of the mean for the metric
+        /// </summary>
+        public double StandardError => ComputeStandardError();
+
+        private SummaryStatistics _statistic;
+
+        internal MetricStatistics()
+        {
+            _statistic = new SummaryStatistics();
+        }
+
+        /// <summary>
+        /// Add another metric
+        /// </summary>
+        /// <param name="metric">The metric being accumulated</param>
+        internal void Add(double metric)
+        {
+            _statistic.Add(metric);
+        }
+
+        private double ComputeStandardDeviation()
+        {
+            double standardDeviation = 0;
+            // Protect against a divid-by-zero
+            if (_statistic.RawCount > 2)
+                standardDeviation = _statistic.SampleStdDev;
+
+            return standardDeviation;
```

Comment:
would this be more readible?
return (_statistic.RawCount <= 2) ? 0 : _statistic.SampleStdDev;

Reply:
Yes, good idea. I went ahead and use this directly on the properties.